testing/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "openzeppelin",
```

Comment:
Should we have this in its own openzeppelin directory? Just for organization purposes?

Reply:
Yes, currently working on it.